pkg/metadata/labels.go

```diff
@@ -133,14 +133,10 @@ func NewInstanceLabeler(instanceMeta metav1.Object) GenericLabeler {
 
 // NewKroMetaLabeler returns a new labeler that sets the OwnedLabel,
 // KroVersion, and ControllerPodID labels on a resource.
```

Comment:
minor: remove ControllerPodID from the comment. I guess this also applies to https://github.com/kro-run/kro/pull/311

Reply:
addressed in https://github.com/kro-run/kro/pull/311